blog/2025-09-30-conda-roadmap-q4-2025.mdx

```diff
@@ -0,0 +1,82 @@
+
+The [Q4 2025 roadmap](https://github.com/orgs/conda/projects/22) highlights the features we are focused on delivering this quarter and early next year. These priorities came directly from community feedback across GitHub, forums, and events, and the roadmap will continue to evolve as new needs emerge. Don’t see something specific on the roadmap? Let us know by [opening an issue](https://github.com/conda/conda/issues), or help us out by [submitting a contribution](https://github.com/conda/conda/blob/main/CONTRIBUTING.md) via a pull request!
+
+
+---
+
+## Q4 2025 Roadmap Priorities
+
+### More Consistency: Input File Handling Refactor
+
+**What we heard:** Conda supports different input files like `environment.yml` and `requirements.txt`, but behavior can feel inconsistent. Switching between formats or mixing them can cause confusion and errors.
+
+**What we are working on:** We are completing a refactor to unify **input file handling** under a plugin-based system. This will make conda’s behavior more consistent and create a cleaner foundation for supporting new formats in the future.
+
+**How you will benefit:** You will see more predictable behavior no matter which file type you start from, reducing surprises when switching between formats. For contributors, this refactor will make it easier to extend conda with new input formats, allowing the ecosystem to evolve more quickly.
+
+---
+
+### Better Reproducibility: Native Lockfile Support
+
+**What we heard:** Reproducing environments exactly can be tricky with `environment.yml`. Teams often share these files expecting identical results, but small version differences across platforms frequently lead to “works on my machine” problems.
+
+**What we are working on:** We are implementing **native lockfile support** in conda. Lockfiles capture the exact packages and builds installed in an environment so they can be recreated reliably elsewhere. This work also includes support for ecosystem formats already in use, such as `conda-lock` and Pixi lockfiles, to ensure consistency across tools.
+
+**How you will benefit:** Using lockfiles to share and recreate conda environments will make them predictable and consistent across development, testing, and production. Environment creation from lockfiles will also be faster since conda can skip solving, which is especially valuable in CI pipelines where environments are frequently rebuilt.
+
+---
+
+### Faster Performance: Sharded Repodata
+
+**What we heard:** Installing or updating packages can be slow because conda downloads a single large index of all available packages, even if you only need a small part. This leads to delays in local workflows, CI pipelines, and on slower networks.
```

Comment:
```suggestion
**What we heard:** Installing or updating packages can be slow because conda downloads a single large index of all available packages, even if you only need a small part. This affects users of [conda-forge](https://conda-forge.org) even more because that package index is especially large . This leads to delays in local workflows, CI pipelines, and on slower networks.
```

Let's make sure to give special notice and respect to users of conda-forge because they are going to be seeing the best performance boosts when this feature is fully implemented.